week5/solutions/cipher/Message.py

```diff
@@ -0,0 +1,28 @@
+import string
+
+from load_words import load_words
+
+class Message(object):
+    def __init__(self, text):
+        self.message_text = text
+        self.valid_words = load_words()
+
+    def get_message_text(self):
+        return self.message_text
+
+    def get_valid_words(self):
+        return self.valid_words[:]
+        
+    def build_shift_dict(self, shift):
+        alphabet = string.ascii_lowercase
+        return dict(list(self.map_alphabet(alphabet, shift).items()) 
```

Comment:
Not sure if it is worth declaring an alphabet variable or better to just pass string.ascii_lowercase and string.ascii_uppercase directly to the calls to map_alphabet.
```
def build_shift_dict(self, shift):
  return dict(list(self.map_alphabet(string.ascii_lowercase, shift).items()) 
  + list(self.map_alphabet(string.ascii_uppercase, shift).items()))
```